docker/local_postgres/Dockerfile

```diff
@@ -3,19 +3,18 @@ FROM postgres:13.6
 ARG PGCLI_VERSION=3.2.0
 
 # Container optimizations
-ARG DEBIAN_FRONTEND=noninteractive
-ENV PIPNOCACHEDIR=1
 ENV PYTHONUNBUFFERED=1
+ENV PIP_NO_CACHE_DIR=1
 ENV PIP_NO_COLOR=1
 
-RUN apt-get update && apt-get -yqq install \
-    python3-boto3 \
-    postgresql-plpython3-13 \
-    python3-pip \
-    libpq-dev \
+RUN apt-get update \
+    && apt-get -yqq install \
+      python3-boto3 \
+      postgresql-plpython3-13 \
+      python3-pip \
+      libpq-dev \
     && apt-get autoremove -y \
     && rm -rf /var/lib/apt/lists/* \
-    && pip3 install -U pip \
```

Comment:
Why remove the pip update step here?

Reply:
It's being combined with the pgcli installation in the following line:

`&& pip3 install -U pip pgcli==${PGCLI_VERSION}`

Not sure about that. Will the -U flag impact pgcli?

Reply:
I checked, `-U` does not affect dependencies with a version number specified.

Reply:
Ahh, my mistake, I missed that!